packages/mdc-data-table/test/foundation.test.ts

```diff
@@ -21,48 +21,11 @@
  * THE SOFTWARE.
  */
 
-import {verifyDefaultAdapter} from '../../../testing/helpers/foundation';
 import {setUpFoundationTest} from '../../../testing/helpers/setup';
 import {attributes, cssClasses, SortValue, strings} from '../constants';
 import {MDCDataTableFoundation} from '../foundation';
 
 describe('MDCDataTableFoundation', () => {
-  it('default adapter returns a complete adapter implementation', () => {
```

Comment:
is removing tests in scope of this PR? Can we remove this tests when we delete default adapters?

This may affect code coverage numbers otherwise.

Reply:
@abhiomkar Initially, we removed these tests because we assumed defaultAdapter would no longer be used (since now adapters need to implement the full interface).

But another test mocks an adapter using the defaultAdapter, so just to get the tests to pass, @kseamon suggested putting back the defaultAdapter for now (we can delete them in a later PR)

Reply:
@kseamon Can you please confirm if this is what you intended. http://b/165316767 suggests that unit tests will be cleaned up along with defaultAdapter code.

Reply:
Sorry - missed this way back when - yes, we need to keep the default adaptors around until the tests are refactored.

The most straightforward thing might be to factor them out into objects that live in the test files.

Reply:
Got it! Moving objects to live in tests SGTM so that the code coverage numbers will be intact.

Since ngwattcos@ left Google, do you want to take a stab at it? You can directly send a CL to us (Copybara will automatically merge the PR).

Let me know if you've any questions!